src/index.ts

```diff
@@ -56,28 +62,40 @@ export interface GetCallOptions extends CallOptions {
   autoCreate?: boolean;
 }
 
-export interface PushConfig {
-  pushEndpoint: string;
-  attibutes?: Map<string, string>;
+export interface Inventory {
+  callbacks?: Array<RequestCallback<string>>;
+  queued?: Array<{}>;
+  bytes: number;
+  nack?: Array<[string, number]>;
+}
+
+export interface Batching {
```

Comment:
This interface already exists https://github.com/googleapis/nodejs-pubsub/blob/master/src/publisher.ts#L44